app/views/registers/index.html.haml

```diff
@@ -87,7 +87,7 @@
                 = sort_link(@search, :register_authority, "Managed by", default_order: :asc, hide_indicator: true)
               %th{role: "columnheader", scope: "col"}
           %tbody
-            - @registers.each_with_index do |register, index|
+            - @registers.select(&:available?).each_with_index do |register, index|
```

Comment:
Can we move the method into the controller?

`@registers = Register.something.select(&:available?)`